docs/sources/github/example-api-responses/sanitized/org_copilot_seats.json

```diff
@@ -0,0 +1,47 @@
+{
+  "total_seats":2,
+  "seats":[
+    {
+      "created_at":"2021-08-03T18:00:00-06:00",
+      "updated_at":"2021-09-23T15:00:00-06:00",
+      "pending_cancellation_date":null,
+      "last_activity_at":"2021-10-14T00:53:32-06:00",
+      "last_activity_editor":"vscode/1.77.3/copilot/1.86.82",
+      "plan_type":"business",
+      "assignee":{
+        "login":"p~IAUEqSLLtP3EjjkzslH-S1ULJZRLQnH9hT54jiI1gbN_fPDYrPH3aBnAoR5-ec6f",
+        "id":"{\"hash\":\"0zPKqEd-CtbCLB1ZSwX6Zo7uAWUvkpfHGzv9-cuYwZc\"}",
+        "node_id":"{\"hash\":\"4YTpKVPu-74u-26baOFgVjym_0UMqn6D8A3cWAjmJ0U\"}",
+        "type":"User",
+        "site_admin":false
+      },
+      "assigning_team":{
+        "id":1,
+        "node_id":"MDQ6VGVhbTE=",
+        "url":"https://api.github.com/teams/1",
+        "html_url":"https://github.com/orgs/github/teams/justice-league",
+        "slug":"justice-league",
```

Comment:
some customers will want slug and html_url dropped, if we can (avoid revealing nature of product teams)